src/accounts/SwitchAccountOverlay.tsx

```diff
@@ -0,0 +1,123 @@
+// Libraries
+import React, {FC, useContext, useState, useEffect} from 'react'
+
+import {CLOUD_URL} from 'src/shared/constants'
+
+// Components
+import {
+  Button,
+  ComponentColor,
+  ComponentSize,
+  ComponentStatus,
+  InputLabel,
+  InputToggleType,
+  Overlay,
+  Toggle,
+} from '@influxdata/clockface'
+
+// Metrics
+import {event} from 'src/cloud/utils/reporting'
+
+import {UserAccountContext} from './context/userAccount'
+
+interface Props {
+  onDismissOverlay: () => void
+}
+
+interface ToggleProps {
+  onClickAcct: (acct: number) => void
+}
+
+const ToggleGroup: FC<ToggleProps> = ({onClickAcct}) => {
+  const {userAccounts, activeAccountId} = useContext(UserAccountContext)
+
+  const [selectedAcct, setSelectedAcct] = useState(activeAccountId)
+
+  // need to change it to a number, so that the equality for
+  // active and isChecked works properly
+  // it is a string because that is how the toggle works
+  const onChange = strValue => {
+    const numAcct = parseInt(strValue)
```

Comment:
```suggestion
    const numAcct = parseInt(strValue, 10)
```

probably safe in 2022, but there's no guarantee that the user's JavaScript engine will parse it in decimal and this would be a really nefarious bug to track down.